plugins/Files/js/components/filetransfers.js

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react'
+import { List } from 'immutable'
+import TransferList from './transferlist.js'
+
+const FileTransfers = ({uploads, downloads, actions}) => {
+	const onCloseClick = () => actions.hideFileTransfers()
+	let uploadlist
+	if (uploads.size > 0) {
+		uploadlist = (
+			<div className="downloads">
+				<h3> Uploads </h3>
+				<TransferList transfers={uploads} />
+			</div>
+		)
+	}
+	let downloadlist
```

Comment:
is the default value `null` or `undefined`? What happens if you try to render `undefined`?